community/modules/scheduler/schedmd-slurm-gcp-v6-controller/outputs.tf

```diff
@@ -21,3 +21,11 @@ output "slurm_bucket_path" {
   description = "Bucket path used by cluster."
   value       = module.slurm_files.slurm_bucket_path
 }
+
+output "instructions" {
+  description = "Post deployment instructions."
+  value       = <<-EOT
+    To SSH to the controller (may need to add '--tunnel-through-iap'):
+      gcloud compute ssh --zone ${var.zone} ${local.controller_hostname} --project ${var.project_id}
```

Comment:
nit. you can make it shorter by outputting

```sh
gcloud compute ssh  ${module. slurm_controller_instance .instances_self_links[0]}
```

Reply:
actual output is probably longer, but it feels less error prone and I like it better. Thanks for the suggestion.

Reply:
ps. I implemented your suggestion if it wasn't clear from my comment.